### PR TITLE
Feature/support multiple networks

### PR DESCRIPTION
**Pull request by stevemul**
Good luck!!

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-openshift_openstack_password=$1
+multinetwork=$1
```
Comment (andywis) [inline deploy.sh]: might be wise to have $1 and $2 in double quotes, and validate

File: deploy.sh [hunk 2]
```diff
@@ -17,22 +18,15 @@ function getPassword() {
...
-  fi
+function setupHeatTemplate() {
+  ansible-playbook ./setup-heat-templates.yaml  --extra-vars "multinetwork=$multinetwork"
 }
 
 function deployHeatStack() {
   openstack stack create -f yaml -t openshift.yaml openshift-$OS_TENANT_NAME \
     -e rhel_reg_creds.yaml \
     -e environment.yaml \
+    --parameter multinetwork=$multinetwork \
```
Comment (andywis) [inline deploy.sh]: all these variables should be in double quotes

File: environment_example.yaml [hunk 3]
```diff
@@ -6,6 +6,10 @@ parameter_defaults:
   s3_access_key: <s3 secret to be used for the OpenShift registry>
   s3_secret_key: <s3 secret to be used for the OpenShift registry>
   s3_bucket_name: <existing s3 bucket to be used for the OpenShift registry>
-  #openshift_version: <major version number eg: 3.5 or 3.6>
+  get_certificates: <whether to get certificates with certbot playbooks>
+  do_upgrades: <whether to perform node patching prior to deployment>
+  #openshift_version: <major version number eg: 3.5 or 3.6 or 3.7 >
   #local_domain_suffix: <local internal domain name - defaults to openstacklocal if not specified>
-  #haproxy_floating_ip: <pre-provisioned floating ip>
+  #controlplane_floating_ip: <pre-provisioned floating ip>
```
Comment (andywis) [inline environment_example.yaml]: Does the HAProxy_floating_ip field get used at all? If not, which *_floating_ip field is used for single-network?

Comment (stevemul) [reply environment_example.yaml]: In short no it doesn't I'll remove that old variable. The control plane and data plane are both needed now as we've effectively split the functionality of the old loadbalancer across two new ones to separate the respective usage, and to ensure control plane maintenance doesn't need to interrupt service to the data plane

**Comment by stevemul** [comment]
All of Andy's changes addressed in subsequent commits and deployments tested successfully.

**Pull request merged**